tests/template_tests/syntax_tests/test_firstof.py

```diff
@@ -81,3 +81,24 @@ def test_firstof13(self):
     def test_firstof14(self):
         output = self.engine.render_to_string('firstof14', {'a': '<'})
         self.assertEqual(output, '<')
+
+    @setup({'firstof15': '{% firstof a b c as myvar %}'})
```

Comment:
It seems to me that 1 test should be sufficient for testing the "as" functionality.